stix_shifter_modules/azure_log_analytics/stix_translation/json/SecurityIncident_from_stix_map.json

```diff
@@ -0,0 +1,79 @@
+{
+    "url": {
+        "fields": {
+            "value": [
+                "IncidentUrl"
+            ]
+        }
+    },
+    "x-ibm-finding": {
+        "fields": {
+            "start": [
+                "FirstActivityTime",
+                "FirstModifiedTime",
+                "TimeGenerated"
+            ],
+            "end": [
+                "ClosedTime",
+                "LastActivityTime",
+                "LastModifiedTime"
+            ],
+            "name": [
+                "IncidentName"
+            ],
+            "time_observed": [
+                "TimeGenerated"
+            ]
+        }
+    },
+    "x-oca-event": {
+        "fields": {
+            "created": [
+                "CreatedTime"
+            ],
+            "provider": [
+                "ProviderName"
+            ],
+            "url_ref": [
+                "IncidentUrl"
+            ],
+            "domain_ref": [
+                "DomainName"
+            ]
+        }
+    },
+    "x-azure-security-incident": {
```

Comment:
Anything that doesn't fit directly into a finding property should be a custom extension of the finding object.